src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/ServiceFactory.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Applications.ComponentBased.Common
+{
+    internal class ServiceFactory : ComponentFactoryBase
+    {
+        private Interop.CBApplication.ServiceLifecycleCallbacks _callbacks;
+        private ComponentBasedApplication _parent;
```

Comment:
It already provide on base class
`public ComponentBasedApplication Parent { get; }`

Reply:
Removed